### ISSUE TITLE
Accessibility: Fix MySQL query editor fastpass issues

### ISSUE THREAD

**Issue by kaydelaney** [grafana/grafana#42718]

**Comment by codeincarnate** [comment]
Closing as this should be fixed by #43981
